bin/main.rb

```diff
@@ -0,0 +1,35 @@
+# Set up game
+puts "Welcome!"
+puts "Player 1, What is your name?"
+@player1 = gets.chomp
+puts "Now, Player 2, What is your name?"
+@player2 = gets.chomp
+puts "Let's play! #{@player1} #{@player2}"
+count = 0
+
+# While nobody has won
+while count < 3
```

Comment:
#### 1. The loop shouldn't depend on a count  , try to implement a variable that triggers the end of the game once it switches from true to false, for example

Reply:
We just put this in here to "mock" the game play.

Reply:
- [ ] Yep, even though it is a mock of the game, and there will be no logic behind it, we are simulating what the user will see on the screen. So, for example, the variable could switch when choosing a given number (maybe 5, or it could be randomized, up to you), since if we use a count it is not a loop, and it is required for this step to implement the loop.

Reply:
Thank you for the clarification! We changed it and submitted another code review. 😸